components/02-molecules/social-links/_social-links.scss

```diff
@@ -0,0 +1,15 @@
+@use '../../01-atoms/atoms';
+@use '../../00-tokens/tokens';
+
+.social-links__list {
+  @include atoms.list-reset;
+
+  display: flex;
+  gap: var(--size-spacing-6);
+}
+
+.social-links__icon {
+  color: var(--color-blue-yale);
+  height: 1em;
+  width: 1em;
```

Comment:
I would probably use rem here, since it's not visually relative to any text around it.

Reply:
Fixed this to now use rem.